File: FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/request/model/PostHelpFeedbackModel.java
/*
 * © 2016-2019 FlowCrypt Limited. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.api.retrofit.request.model;

import com.flowcrypt.email.api.retrofit.request.api.PostHelpFeedbackRequest;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * The model of {@link PostHelpFeedbackRequest}.
 *
 * @author DenBond7
 * Date: 30.05.2017
 * Time: 12:42
 * E-mail: DenBond7@gmail.com
 */

public class PostHelpFeedbackModel extends BaseRequestModel {

  @Expose
  private String email;

  @SerializedName("message")
  @Expose
  private String msg;

  public PostHelpFeedbackModel(String email, String msg) {
    this.email = email;
    this.msg = msg;
  }

  public String getEmail() {
    return email;
  }

  public String getMsg() {
    return msg;
  }
}

File: FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/request/model/PostHelpFeedbackModel.kt
/*
 * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
 * Contributors: DenBond7
 */

package com.flowcrypt.email.api.retrofit.request.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author Denys Bondarenko
 */
data class PostHelpFeedbackModel(
  @Expose val email: String,
  @Expose val logs: String? = null,
  @Expose val screenshot: String? = null,
  @SerializedName("message") @Expose val msg: String
) : RequestModel
